src/main/java/de/tum/in/www1/artemis/service/plagiarism/PlagiarismDetectionService.java

```diff
@@ -38,6 +39,9 @@ public class PlagiarismDetectionService {
 
     private final PlagiarismResultRepository plagiarismResultRepository;
 
+    @Value("${artemis.plagiarism-checks.plagiarism-results-limit}")
```

Comment:
This will need adjustments on prod, staging, and the test servers.
Can you please do the following:
- Add a note to the PR description, that deploying is currently not possible due to this
- If possible create a PR for the [artemis-ansible-collection](https://github.com/ls1intum/artemis-ansible-collection) that adds this value to the config files
- Tell us in #artemis-deployment-wg on Slack what the recommended values for the different Artemis servers should be, so we can roll them out to the instances eventually

Reply:
@Hialus PR ready: https://github.com/ls1intum/artemis-ansible-collection/pull/51